### PR TITLE
Sales Links / replaced images with table and code

### PR DESCRIPTION
**Pull request by Catografix**
Changes sales@auth0.com  to https://auth0.com/?contact=true

In SLA, replaced images with table and code.

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/sla.md [hunk 1]
```diff
@@ -27,11 +27,19 @@ Subscriber shall receive no SLCs in connection with any unavailability of the Su
...
+| < 99.9% - >= 99.0% | 3 |
+| < 99.0%  - >= 95.0% | 7 |
+| < 95.0% | 15 |
 
 Availability is calculated as follows: 
 
-![](https://docs.google.com/drawings/d/1DJtyLsHJrZZySFWwtx0hEs3wTBbiJJ-zMrDjz0oOUFI/pub?w=549&h=100)
+```
+(Total Time) - (Non-Permitted Downtime) - (Permitted Downtime)
+--------------------------------------------------------------  X 100
```
Comment (Catografix) [inline articles/sla.md]: @ntotten Do you like this solution?

**Comment by Catografix** [comment]
Preview OK: https://auth0-docs-content-pr-1932.herokuapp.com/docs/sla

File: articles/sla.md (hunk 1 above)
Comment (ntotten) [inline articles/sla.md]: Make an SVG: https://viereck.ch/latex-to-svg/

**Pull request merged**